docker/validator-dynamic/validator-dynamic.Dockerfile

```diff
@@ -16,11 +16,8 @@ FROM toolchain AS config_builder
 
 COPY . /libra
 
-RUN cargo build --release -p libra-node -p cli -p config-builder -p safety-rules && cd target/release && rm -r build deps incremental
```

Comment:
if you don't need to build, do you need those rust setup above?

Reply:
Good question. I actually expect that to be a no-op. Let me re-run it and try to minimize more.